Userland/Utilities/xxd.cpp

```diff
@@ -65,10 +123,10 @@ static void print_line_hex(Bytes line, size_t line_length_config, size_t group_s
     out(" ");
 }
 
-static void print_line_little_endian_hex(Bytes line, size_t line_length_config, size_t group_size, bool uppercase)
+static void print_line_little_endian_hex(Bytes line, size_t line_length_config, size_t group_size, bool uppercase, bool colorize_output)
```

Comment:
Consider using an enum class instead of a bool: https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#right-8 (also a bit further up). Especially if there are two bool parameters right next to each other, it's easy to accidentally swap them. With enum classes, the type system prevents that type of error.